### ISSUE TITLE
Uploading Artifacts to S3 Endpoint

### ISSUE THREAD

**Issue by alex-g-tejada** [mlflow/mlp-regression-template#57]
Is there a way to change the artifacts location other than local? I set MLFLOW_S3_ENDPOINT_URL with user/pass in the jupyternotebook but every run sends the model to metadata/mlflow/mlartifacts/..

**Comment by jinzhang21** [comment]
Yes. You can specify `artifact_location` in the profile config. For example here: https://github.com/mlflow/mlp-regression-template/blob/main/profiles/local.yaml#L4

**Comment by alex-g-tejada** [comment]
Thanks @jinzhang21 I did try to replace this with a s3 url, but It still stores the artifacts locally

**Comment by bvarjavand** [comment]
Hi,
I've been reading source code to try to figure out how to resolve this.

I tried adding overrides in the local.yaml for the aws public and private keys but that prevents the pipeline from running

**Comment by jinzhang21** [comment]
Thanks for the feedback @alex-g-tejada and @bvarjavand ! We'll investigate this in the coming weeks.
